src/Event/Http/FpmHandler.php

```diff
@@ -282,4 +290,33 @@ private function getResponseHeaders(ProvidesResponseData $response): array
     {
         return array_change_key_case($response->getHeaders(), CASE_LOWER);
     }
+
+    /**
+     * Log a message to Cloudwatch, this is specific for FPM related errors and will include AWS Request IDs for the
+     * current Lambda invocation and API Gateway (where applicable) to aid debugging any FPM-related issues.
+     */
+    private function logToCloudwatch(string $message): void
+    {
+        echo trim($message) . ' ' . json_encode($this->getLogContext()) . PHP_EOL;
+    }
+
+    /**
+     * Returns an array with AWS Request IDs for the current Lambda invocation and API Gateway (where applicable)
+     *
+     * @return string[]|array
+     */
+    private function getLogContext(): array
+    {
+        $logContext = [];
+
+        if ($this->lambdaRequestId) {
+            $logContext['lambdaRequestId'] = $this->lambdaRequestId;
```

Comment:
We could use the same terminology as existing Lambda CloudWatch logs:

```suggestion
            $logContext['requestId'] = $this->lambdaRequestId;
```

<img width="1189" alt="image" src="https://user-images.githubusercontent.com/720328/108326604-b9d4de00-71ca-11eb-80ca-ba51ef6edafb.png">

Reply:
My thinking here was that as there are potentially two different request IDs in the log here (if invoked via API Gateway) that it would make sense to be more explicit to avoid confusion, but if you'd rather remain with the existing terminology, I've no issue with that

Reply:
Right, I understand but yep I think it's better to rather stay in line with AWS naming.